### PR TITLE
Cndit 1535

### PR DESCRIPTION
**Pull request by ndduc01**
Update kafka config and logic, stabilizing the flow when it handles high volume of data

### CODE REVIEW

File: data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaEdxLogConsumer.java
Comment (ssathiah) [inline data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaEdxLogConsumer.java:1]: You can remove the commented code if it's not needed.

File: data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaHandleLabConsumer.java
Comment (ssathiah) [inline data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaHandleLabConsumer.java:1]: You can remove the commented code if it's not needed.

File: data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaPublicHealthCaseConsumer.java
Comment (ssathiah) [inline data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaPublicHealthCaseConsumer.java:1]: You can remove the commented code if it's not needed.

**Review by ssathiah — APPROVED** [review]
Looks good to me.

File: data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaEdxLogConsumer.java
Comment (ndduc01) [reply data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaEdxLogConsumer.java:1]: I left it there for references for now

File: data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaPublicHealthCaseConsumer.java
Comment (ndduc01) [reply data-processing-service/src/main/java/gov/cdc/dataprocessing/kafka/consumer/KafkaPublicHealthCaseConsumer.java:1]: I left it there for references for now

**Pull request merged**